packages/amplify-cli-core/src/__tests__/hooks/hooksExecutor.test.ts

```diff
@@ -81,6 +81,7 @@ describe('hooksExecutioner tests', () => {
   });
   afterEach(() => {
     HooksMeta.releaseInstance();
+    delete process.env.AMPLIFY_CLI_DISABLE_SCRIPTING_FEATURES;
```

Comment:
this is unit test. 
added extra delete to make sure it's purged in case test fails.